setup.py

```diff
@@ -107,14 +107,13 @@ def run(self):
 setup(
     name="responder",
     version=about["__version__"],
-    description="A sorta familiar HTTP framework.",
+    description="A familiar HTTP Service Framework for Python.",
     long_description=long_description,
     long_description_content_type="text/markdown",
     author="Kenneth Reitz",
     author_email="me@kennethreitz.org",
     url="https://github.com/kennethreitz/responder",
     packages=find_packages(exclude=["tests"]),
-    entry_points={"console_scripts": ["responder=responder.cli:cli"]},
```

Comment:
Dear @kennethreitz. While I am removing this reference for now, it could be sweet to add it back later. Is the CLI a vaporware feature, or did you just forget to add it to the repository?

Reply:
vaporware apparently